### ISSUE TITLE
subtitles not getting embed

### ISSUE THREAD

**Issue by thaynos** [jely2002/youtube-dl-gui#65]
ive mentioned this bug before in #53 but it hasnt been fixed in the new version. auto gen subtitles arent embed and even videos with their own subtitles arent embed.
also the videos with their own subtitles werent embed in the previous version either.
example
![image](https://user-images.githubusercontent.com/83282129/118609109-7b4fbf00-b7cb-11eb-9aac-d4a56e71b36e.png)

example video link
https://www.youtube.com/watch?v=iBrJIwAQQbE
![Screenshot (245)](https://user-images.githubusercontent.com/83282129/118609997-60ca1580-b7cc-11eb-8f81-42ea67950495.png)

**Comment by jely2002** [comment]
I am unable to replicate this issue on my side, the subtitles always get embedded and can be accessed in VLC.
![image](https://user-images.githubusercontent.com/20154900/118636047-8c50fe00-b7d4-11eb-92d8-ab94c33a78a8.png)
Have you actually enabled subtitles on the video? This can be done by clicking on the subtitle icon, there should not be a strike through the icon. Like this:
![Screenshot 2021-05-18 122954](https://user-images.githubusercontent.com/20154900/118636247-c3bfaa80-b7d4-11eb-8507-87931ea01fbb.png)

Please let me know if you managed to figure it out.

**Comment by thaynos** [comment]
yes i always set it as the icon without the cross. i even tried the option with the cross recently to see if it will work but it dint.

**Comment by jely2002** [comment]
Hi thaynos,
This should now be fixed in v2.1.2. Please let me know if this is the case, if not I will reopen the issue.

**Comment by breakstring** [comment]
Bug repro in v2.2.2,it was a .VTT file in download folder during the downloading process. 
But when the task finished, the .VTT file was gone, and the video no subtitle embedded.

**Comment by jely2002** [comment]
Could you provide me with some more info, such as the url and the settings you tried downloading it with?
Can you see the subtitles when you open it in VLC media player and navigate to the subtitles menu?

**Comment by breakstring** [comment]
Oops...So sorry, looks it's the media player issue(subtitles compatibility). 
1. I used ['QQPlayer'](https://player.qq.com/) yesterday, which is very popular in China, but it cannot show the embedded subtitles(video 1). It's my bad, I forgot check the subtitle information in video 1.
2. I converted the .vtt to .ass then embedded it to the video again with ffmpeg by myself. QQPlayer can show the subtitles which I embedded handmade.(video 2)
3. I installed the VCL player today, and it can show subtitles in both video1 & video2. 
4. I just checked the subtitle information with ffprobe. Yes, all these two videos contain subtitles. video1 with webvtt and video2 with ass format.
